### DISCUSSION TITLE
Playing tracks directly in the browser

### DISCUSSION THREAD

**Discussion by devklick** [JohnnyCrazy/SpotifyAPI-NET#815] [Q&A] [answered]
Is it possible to play tracks/episodes in a browser using this SDK? For example, I want to create a Blazor web app; can I play tracks in that app, or can I only control other devices connected to Spotify (desktop app, dobile app)?

From my limited usage with it so far, it seems like it only controls Spotify clients on other devices (desktop app, mobile app). I havent found anything concrete in either the SpotifyAPI-Net docs and examples, nor the official Spotify API docs, to state whether what I'm trying to do is at all possible. It seems strange for the API to _not_ have this functionality, so hopefully I've just missed something.

I've noticed that there's a [JS Web Playback SDK](https://developer.spotify.com/documentation/web-playback-sdk/quick-start/) which looks like it does what I'm after; creates and controls a player in the browser rather than controlling a Spotify client. Was hoping to avoid writing vanilla JS, though.

**Comment by JohnnyCrazy** [comment, marked as answer]
No, as this wrapper is only for the Web API, not the playback SDK.
